asynchronous-authorization/langchain-next-js/package-lock.json

```diff
@@ -78,7 +80,7 @@
     },
     "node_modules/@alloc/quick-lru": {
       "version": "5.2.0",
-      "resolved": "https://registry.npmjs.org/@alloc/quick-lru/-/quick-lru-5.2.0.tgz",
+      "resolved": "https://a0us.jfrog.io/artifactory/api/npm/npm/@alloc/quick-lru/-/quick-lru-5.2.0.tgz",
```

Comment:
apologies here -> we will need to regenerate all lock files. My machine was unfortuantely targeting the default Artifactory registry when i was testing these.

We'll need to regenerate anyways when we update to target released packages, but let's try to:
```
rm -rf node_modules/ package-lock.json
npm install --registry https://registry.npmjs.org/
```

When installing next time...

Reply:
(no Artifactory packages should be referenced in these public repos)